### PR TITLE
chore(*): bump version to v0.12.0

### PR DESCRIPTION
**Pull request by Mossaka**
prepare for the release

### CODE REVIEW

File: containerd-shim-spin/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "containerd-shim-spin-v2"
-version = "0.10.0"
+version = "0.12.0"
 authors = ["DeisLabs Engineering Team"]
```
Comment (kate-goldenring) [inline containerd-shim-spin/Cargo.toml:4]: Should we call this the "SpinKube Engineering Team" now?

**Review by kate-goldenring — APPROVED** [review]
LGTM

File: containerd-shim-spin/Cargo.toml (hunk 1 above)
Comment (Mossaka) [reply containerd-shim-spin/Cargo.toml]: done

**Review by devigned — APPROVED** [review]
lgtm!

**Pull request merged**